docs/guides/requests_storage.md

```diff
@@ -0,0 +1,137 @@
+---
+id: requests-storage
+title: Requests Storage
+---
+
+The Apify SDK has several requests storage types that are useful for specific tasks. The requests are stored either in a run memory (applies to Request List only, see below), on local disk to a directory defined by the
```

Comment:
request storage, request is an adjective here

Reply:
request list is also normally persisted to disk, unless user chooses `null` as request list name

Reply:
fixed removed part about run memory